client/packages/electron/src/discovery/ServerDiscovery.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {
+  useElectronClient,
+  InlineSpinner,
+  BaseButton,
+  AlertIcon,
+  Box,
+  Typography,
+  useTranslation,
+  frontEndHostDisplay,
+} from '@openmsupply-client/common';
+
+import { ServerDiscoveryLayout } from './ServerDiscoveryLayout';
+
+export const ServerDiscovery = () => {
```

Comment:
I have to confess that I was confused when confronted by this screen. Took a while to realise that the server button was a button, and that I was supposed to select a server.

I think the screen would be better if it was much simpler, with instructions to select a server and an explanation what that meant.

Would you mind if I have a play once this PR is merged? I can put some ideas together and see what y'all think?

Reply:
I agree. I think it would be confusing to users. I just wanted to set it all up so that it can be easily worked (ability to use existing packages etc..).

> Would you mind if I have a play once this PR is merged? I can put some ideas together and see what y'all think?

That would be awesome, thanks. I think the dev environment should make it pretty easy to update UI of the bundled bit of the app